common/djangoapps/pipeline_mako/templates/mako/inline_js.html

```diff
@@ -1,9 +1,10 @@
+<%page expression_filter="h"/>
 <script \
-% if async:
+% if context.get('async', default=None):
 async \
 % endif
 if defer:
 defer \
 type="text/javascript" charset="utf-8">
-    ${source | safe}
+    ${source | n, decode.utf8}
```

Comment:
I think that we can use here now:
```suggestion
    ${source}
```
since now, everything is safe by default but I am not sure. What do you think @jmbowman?